homeassistant/core.py

```diff
@@ -510,16 +511,35 @@ def create_task(self, target: Coroutine[Any, Any, Any]) -> None:
         self.loop.call_soon_threadsafe(self.async_create_task, target)
 
     @callback
-    def async_create_task(self, target: Coroutine[Any, Any, _R]) -> asyncio.Task[_R]:
+    def async_create_task(
```

Comment:
I'm on the fence if this should be `async_create_background_task` instead to make it easier to refactor later in case we need to change the behavior so we don't end up with a lot of future branching.